### ISSUE TITLE
Unnecessary styles being passed to custom render functions

### ISSUE THREAD

**Issue by philraj** [malte-wessel/react-custom-scrollbars#215]
I'm trying to create some custom tracks and thumbs for the scrollbar (via `renderTrackHorizontal`, etc.), but for some reason react-custom-scrollbars passes in `width:100%` to the thumb and `width: 6px` to the track as inline styles. It's a little odd that I need to set `!important` on my CSS just to have them apply when it seems the intended use of the custom render functions is to provide a blank slate.

[edit] Alternatively, if the idea is to provide some sane defaults that are meant to be overridden by merging with a user-defined styles object, and supporting CSS overrides without `!important` is never going to be a goal, it could at least pass in some helpful defaults like `height: 100%; top: 0; right: 0` for the vertical track, and similar for the horizontal.

**Comment by bristoljon** [comment]
Yeah I'm hitting this as well. Some styles are dynamic based on content height and width etc. so you would expect the thumb height to be passed in for the vertical scrollbar but it shouldn't be setting the width of it. It's the same with horizontal.
